gap/planar.gi

```diff
@@ -32,69 +32,90 @@
 
 InstallMethod(PlanarEmbedding, "for a digraph", [IsDigraph],
 function(D)
-  if IsEmptyDigraph(D) or DigraphNrVertices(D) < 3 then
-    return [];
-  elif HasIsPlanarDigraph(D) and not IsPlanarDigraph(D) then
+  local C;
+  C := DigraphMutableCopy(D);
+  DigraphRemoveAllMultipleEdges(C);
+  DigraphRemoveLoops(C);
+  if IsEmptyDigraph(C) or DigraphNrVertices(C) < 3 then
+    return OutNeighbors(C);
+  elif HasIsPlanarDigraph(C) and not IsPlanarDigraph(C) then
```

Comment:
This will always be false because C is mutable and hence Has no attributes. These conditions should probably be applied before C is created

Reply:
Thanks for pointing this out; I've fixed this now.